# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsINode.h
# Commit: edcf770a3a1d
# Full Hash: edcf770a3a1df7b6d618062eb7cdbde9596adbad
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-03-26 04:11:24
# Regressor Bug: 1867058
# File Overlap Count: 1
# Description:
#   Bug 1867058 - Part 8: Update Selection to support across shadow dom selection r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195310
# ==============================================================================

diff -r d665bdcc228a -r edcf770a3a1d dom/base/nsINode.h
--- a/dom/base/nsINode.h	Mon Mar 25 13:40:58 2024 +0000
+++ b/dom/base/nsINode.h	Mon Mar 25 13:40:59 2024 +0000
@@ -1629,7 +1629,7 @@
    * not in same subtree, this returns the root content of the closeset subtree.
    */
   MOZ_CAN_RUN_SCRIPT nsIContent* GetSelectionRootContent(
-      mozilla::PresShell* aPresShell);
+      mozilla::PresShell* aPresShell, bool aAllowCrossShadowBoundary = false);
 
   nsINodeList* ChildNodes();
 